packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/FlutterApiErrorUtils.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.amplify.amplify_api
+
+import android.os.Handler
+import android.os.Looper
+import androidx.annotation.NonNull
+import com.amplifyframework.api.ApiException
+import io.flutter.plugin.common.MethodChannel
+
+class FlutterApiErrorUtils {
```

Comment:
Avoid `Utils` in all names.

This looks like a factory, though, since it builds errors? `FlutterApiErrorFactory`?